VTUL/dlp-access
# Multimedia component for homepage, plus a few random edits.

**JIRA Ticket**: (link) (:star:)
https://webapps.es.vt.edu/jira/browse/LIBTD-2108

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)
 https://xd.adobe.com/view/02fc158b-3aaa-4619-671b-65852af58b64-74cd/

# What does this Pull Request do? (:star:)
Creates MediaSection component, adds it to Homepage.js, and creates .css file for the component; A few other small edits to the app are included.

# What's the changes? (:star:)
App.js - removes container padding because the side margins were too wide for mobile
Footer.js - changes the library's phone number
HomePage.css - adds padding to bottom of section.
SearchBar.css - fixes a small problem with Safari rendering.
MultimediaSection.css - styles the component
HomePage.js - edits add the Multimedia component to the page.
MultimediaSection.js - file for the component

# How should this be tested?
So, I hope I did this correctly, but I pushed an dummy iawa.json file that can be used to test this component since no data exists for it in the real iawa.json file. But if you add it to your local site_data folder, then start the app with REACT_APP_REP_TYPE=IAWA REACT_APP_CONFIG_PATH='http://localhost:3000/site_data' npm start, then you should be able to see the component render with the example data that I've used.  At mobile size, the component should reorganize with the video on top and the text block below.

# Additional Notes:
LIBTD-2108

# Interested parties
@yinlinchen 

(:star:) Required fields
